staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5276,7 +5277,8 @@ type DownwardAPIVolumeSource struct {
 	// +optional
 	Items []DownwardAPIVolumeFile `json:"items,omitempty" protobuf:"bytes,1,rep,name=items"`
 	// Optional: mode bits to use on created files by default. Must be a
-	// value between 0 and 0777. Defaults to 0644.
+	// decimal equivalent of octal value between 0 and 0777.
+	// Defaults to octal value 0644.
```

Comment:
Git hub is not letting me fix this up properly ...
My suggestion is below but you might have to do this on your branch locally 

```suggestion
        // Optional: mode bits used to set permissions on created files by default. Must be an
	// octal value between 0 and 0777. Defaults to 0644.
```